src/tcgwars/logic/impl/gen4/LegendsAwakened.groovy

```diff
@@ -2894,276 +3192,300 @@ public enum LegendsAwakened implements LogicCardInfo {
-        return levelUp (this, from:"Gliscor", hp:HP110, type:FIGHTING, retreatCost:0) {
+        return levelUp (this, from:"Gliscor", hp:HP110, type:F, retreatCost:0) {
           weakness W
           resistance F, MINUS20
           pokePower "Shoot Poison", {
-            text "Once during your turn , when you put Gliscor LV. from your hand onto your Active Gliscor, you may choose 1 of the Defending Pokémon. That Pokémon is now Paralyzed and Poisoned."
+            text "Once during your turn (before your attack), when you put Gliscor LV.X from your hand onto your Active Gliscor, you may choose 1 of the Defending Pokémon. That Pokémon is now Paralyzed and Poisoned."
             actionA {
+              // TODO
             }
           }
           move "Night Slash", {
             text "60 damage. You may switch Gliscor with 1 of your Benched Pokémon."
             energyCost F, C
             attackRequirement {}
             onAttack {
-              damage 0
+              damage 60
+              switchYourActive(may: true)
             }
           }
         };
       case MAGNEZONE_LV_X_142:
-        return levelUp (this, from:"Magnezone", hp:HP140, type:METAL, retreatCost:4) {
+        return levelUp (this, from:"Magnezone", hp:HP140, type:M, retreatCost:4) {
           weakness R
           resistance P, MINUS20
           pokePower "Electric Trans", {
-            text "As often as you like during your turn , you may move a or Energy attached to 1 of your Pokémon to another of your Pokémon. This power can’t be used if Magnezone is affected by a Special Condition."
+            text "As often as you like during your turn (before your attack), you may move a Lightning or [M] Energy attached to 1 of your Pokémon to another of your Pokémon. This power can't be used if Magnezone is affected by a Special Condition."
             actionA {
             }
           }
           move "Cyber Shock", {
-            text "80 damage. Energy attached to Magnezone. The Defending Pokémon is now Paralyzed."
-            energyCost L, M, L, M
+            text "80 damage. Discard a [L] Energy and a [M] Energy attached to Magnezone. The Defending Pokémon is now Paralyzed."
+            energyCost L, M
             attackRequirement {}
             onAttack {
-              damage 0
+              damage 80
+              discardSelfEnergyAfterDamage L, M
+              applyAfterDamage PARALYZED
             }
           }
         };
       case MESPRIT_LV_X_143:
-        return levelUp (this, from:"Mesprit", hp:HP090, type:PSYCHIC, retreatCost:1) {
+        return levelUp (this, from:"Mesprit", hp:HP090, type:P, retreatCost:1) {
           weakness P
           move "Healing Look", {
             text "Remove 3 damage counters from each of your Benched Pokémon."
-            energyCost ()
-            attackRequirement {}
-            onAttack {
+            attackRequirement {
               assert my.bench : "Your bench is empty"
```

Comment:
I think assert my.bench.findAll{it.numberOfDamageCounters} would be better here, as it prevents accidentally using this move to no effect.